spec.html

```diff
@@ -37871,6 +37872,7 @@ <h1>AtomicLoad ( _typedArray_, _index_ )</h1>
         <h1>ByteListBitwiseOp( _op_, _xBytes_, _yBytes_ )</h1>
         <p>The abstract operation ByteListBitwiseOp takes arguments _op_ (a read-modify-write modification function), _xBytes_ (a List of byte values), and _yBytes_ (a List of byte values). The operation atomically performs a bitwise operation on all byte values of the arguments and returns a List of byte values. It performs the following steps when called:</p>
         <emu-alg>
+          1. Assert: _op_ is ~&amp;~, ~|~, or ~^~.
```

Comment:
The preamble's description of `_op_` should be modified likewise (which might take care of Issue #1989).